data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,319 @@ mission "Wanderers: Pre-Colony"
+				`	(Go over and investigate.)`
+					goto investigate
+				`	(Leave and explore this part of space.)`
+					decline
+			label investigate
+			`	The argument seems to be breaking up, with unhappy captains heading back to their ships. You see a few Tempests powering up for departure.`
+			`	You walk up to the Wanderer they were all shouting at.  He looks at you warily as you greet him in Hai. He is briefly startled and cocks his head and regards you for a moment and then says, "Ah. You are the [lost, lonely] one from from the worlds beyond the Hai." He hold up a taloned hand and continues before you can say anything, "This spaceport isn't even begun. I don't have any Thunderheads. I don't have any parts. I don't have any fuel. If you need something, you have to go back to <destination>. I don't know when it will get better because everyone is [dithering, flying around] and either exploring or getting killed and no one is [available, willing] to do transport work."`
+			choice
+				`	"Tell me what you need and I'll handle it."`
+					goto handle
+				`	"I guess I'll just leave, then."`
+					decline
+			label handle
+			`	"[Truly? Honestly?]" he asks. When you nod your head he hands you a data chip and says, "Take this to Vara K'chrai and they will make the arrangements. We need construction equipment, materials and workers and I can't even request them directly since communications don't work through the wormhole.`
+			`	You turn to go as group of Wanderers with surveying equipment walk up. They are all dressed in hard hats and safety gear. The construction supervisor you've been speaking to pulls up a hologram of the site to discuss with the surveyors. As you walk off, the translator just picks up. "The Eye opens and we are getting help from the [alien, stranger] and not our own flock."`
+				accept
+
+	on complete
+		conversation
+			`As you land at the spaceport, you're sure you see a few of the ships you saw on <planet> with various outfits stacked for installation. It's hard to find anyone who speaks Hai, but you find your way to the main spaceport office and hand over the chip. The person at the desk chitters excitedly and calls someone over. They quickly enter information into their systems and then hand the chip back to you. After a few moments of confusion where you end up making your Hai ear gesture, they call up an app and speak into it. Your translator picks up "Job board. Job board." Hopefully everything has been booked for you.`
+
+mission "Wanderers: Spera Anatrusk Outfitter 2a"
+	landing
+	name "Construction workers to <planet>"
+	description "Now that the harvest is over, bring workers from <stopovers> to <destination> to build the new outfitter. Payment is <payment>."
+	blocked "The building supervisor at <destination> has [requisitioned, hired] more workers than you can carry. You will need <capacity> to transport these workers."
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Vara Rakak"
+	passengers 75
```

Comment:
That's a lot of passengers for a campaign mission.

Reply:
Can you provide guidance on what the numbers should be for non-escort missions? I chose the larger numbers of people and cargo volumes to reflect the scale of the work being done. It seems I have a couple of choices if I keep the volumes: breaking them up into more smaller missions or making everything an escort mission. I'm not averse to making changes, but I'm curious what numbers you expect. Given this point in the Wanderer campaign and how lucrative Wanderer space is, I generally have much more than this capacity.

Reply:
Maybe check if player have enough passenger space and offer an escort job as alternative if they don't have enough space?
iirc there's `"passengers"` condition of somesort in the savefile which you might be able to use to check during on offer.

Reply:
I've adjusted the cargo sizes already. The passenger numbers are well within what is offered in Wanderer space. I'd like to avoid an escort mission at this point to keep variety in the chain, and I have explicit escort missions later.  That is an interesting suggestion, though, and would stretch my mission coding skills.